### PR TITLE
fix [GitHubHacktoberfest] service test

### LINKED ISSUE #4298: Hacktoberfest service test failing
:clock11: **When did the problem start?**
~Nov 3rd

:camera: **Live badge**
https://img.shields.io/github/hacktoberfest/2019/snyk/snyk

:wrench: **Is the live badge working?**
Yes

:link: **CircleCI link**
https://circleci.com/gh/badges/daily-tests/705

:beetle: **Stack trace**

```
GithubHacktoberfestCombinedStatus [live] GitHub Hacktoberfest combined status [ GET /badges/shields.json ]
  ValidationError: message mismatch: "value" with value "127 PRs" fails to match the required pattern: /^[0-9]+ open issues?(, [0-9]+ PRs?)?(, [0-9]+ days? left)?$/
```

:bulb: **Possible solution**
The regex validator used for the service tests is expecting there to be at least 1 issue with the `hacktoberfest` label. I went through our issues the other day and removed those labels, and since the service test is using this repo, the test started failing. 

We just need to update the regex

### PR DESCRIPTION
**Pull request by calebcartwright**
Fixes #4298 

I also removed our Hacktoberfest badge from our readme now that we've transitioned into November. 

Related aside, while working on this I find myself wondering what this badge should look like outside October/Hacktoberfest. It looks like it will only display the `is over!` message if there are no `hacktoberfest`-labeled issues, there were no PRs opened during October, and October has ended.

Would it make sense to display `is over!` after Hacktoberfest has ended regardless?

### CODE REVIEW

**Comment by paulmelnikow** [comment]
> Would it make sense to display `is over!` after Hacktoberfest has ended regardless?

Yea, I think so. Maybe something like **is over! (18 pull requests opened)**?

**Comment by calebcartwright** [comment]
> > Would it make sense to display `is over!` after Hacktoberfest has ended regardless?
> 
> Yea, I think so. Maybe something like **is over! (18 pull requests opened)**?

I like that! That makes the badge more meaningful outside of October IMO. You could see one day down the road something like...

-orange)
-orange)
-orange)
-orange)

As part of the leadup to Octoberfest/trying to do more than last year/etc.

File: services/github/github-hacktoberfest.service.js [hunk 1]
```diff
@@ -120,7 +120,14 @@ module.exports = class GithubHacktoberfestCombinedStatus extends GithubAuthV4Ser
       // We want to show "1 day left" on the last day so we add 1.
       daysLeft = moment('2019-11-01 12:00:00 Z').diff(moment(), 'days') + 1
     }
-
+    if (daysLeft <= 0) {
+      return {
+        message: `is over! (${metric(contributionCount)} ${maybePluralize(
```
Comment (paulmelnikow) [inline services/github/github-hacktoberfest.service.js]: > `metric(contributionCount)`

So much optimism I had when I first wrote that 😀

Comment (calebcartwright) [reply services/github/github-hacktoberfest.service.js]: 😄 Gotta supprot those `22Y PRs`

That does have me wondering which repos got the most PRs during Hacktoberfest

Comment (paulmelnikow) [reply services/github/github-hacktoberfest.service.js]: I bet @MattIPv4 knows the answer to that. Though he may also be needing a big break right about now 😁

Comment (MattIPv4) [reply services/github/github-hacktoberfest.service.js]: Ask me again next week and I'll be able to get an answer for you. We don't have the full data yet as not everything finishes and settles down until the 8th :) (1st + 7 day review period)

**Pull request merged**